notebooks/official/matching_engine/sdk_matching_engine_for_indexing.ipynb

```diff
@@ -98,171 +98,172 @@
     {
```

Comment:
<p>A super tiny nit, change to BigQuery </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1249/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>